pkg/upstream/cluster/cluster_manager.go

```diff
@@ -173,6 +201,7 @@ func (cm *clusterManager) AddOrUpdateClusterAndHost(cluster v2.Cluster, hostConf
 		UpdateClusterResourceManagerHandler(oc, nc)
 		CleanOldClusterHandler(oc, nc)
 		NewSimpleHostHandler(nc, hostConfigs)
+		TransferClusterHostStatesHandler(oc, nc)
```

Comment:
`NewSimpleHostHandler` will be called in `NewSimpleHostHandler` , so it is not necessary to call `TransferClusterHostStatesHandler`.

Reply:
`NewSimpleHostHandler(nc, hostConfigs)` just is `nc` but not transfer states from `oc` to `nc`